package.json

```
@@ -58,8 +58,7 @@
     "url": "https://github.com/balena-io/balena-preload/issues"
   },
   "scripts": {
-    "lint": "balena-lint -e js --typescript lib",
-    "lint-fix": "balena-lint -e js --typescript --fix lib",
+    "lint": "balena-lint -e js --typescript --fix lib",
     "lint-python": "flake8 src/preload.py",
     "test": "npm run lint",
```

Given that we are so serious about linting that it's the only thing we care about in our automated testing, :-) we might as well make sure that the build fails if the linter produces uncommitted changes, as follows:
```suggestion
    "test": "npm run lint && catch-uncommitted --skip-node-versionbot-changes",
```
Plus adding `"catch-uncommitted": "^2.0.0",` to the `devDependencies` section of `package.json`.
Ref: https://github.com/balena-io-modules/catch-uncommitted/
I expect this would work, but I am not sure...